src/Raven.Server/Documents/Subscriptions/SubscriptionConnectionsState.cs

```diff
@@ -27,7 +27,6 @@ public class SubscriptionConnectionsState : SubscriptionConnectionsStateBase<Sub
         private readonly SubscriptionStorage _subscriptionStorage;
         public DocumentDatabase DocumentDatabase => _subscriptionStorage._db;
         private IDisposable _disposableNotificationsRegistration;
-        public string LastChangeVectorSent;
         public string PreviouslyRecordedChangeVector;
```

Comment:
where did the `LastChangeVectorSent` go?

Reply:
its in teh base class: `AbstractSubscriptionConnectionsStateBase`